dynamo-event-to-datacite-request/src/main/java/no/unit/nva/events/examples/DestinationsEventConsumer.java

```diff
@@ -0,0 +1,37 @@
+package no.unit.nva.events.examples;
+
+import com.amazonaws.services.lambda.runtime.Context;
+import no.unit.nva.events.handlers.DestinationsEventBridgeEventHandler;
+import no.unit.nva.events.models.AwsEventBridgeDetail;
+import no.unit.nva.events.models.AwsEventBridgeEvent;
+import nva.commons.utils.JacocoGenerated;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Demo class. Will be deleted.
```

Comment:
Check documentation of EventProducer

Reply:
I think it is fine to include a example classes, but would be nice to have these reside under `src/test/java` or if this was a seperate project, even an example module. 

It clearly shows how to use the abstraction which is big :heart:

Reply:
Placed in new module.